datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -310,26 +324,32 @@ private static void initS3() {
         }
 
         AmazonS3ClientBuilder amazonS3ClientBuilder = AmazonS3ClientBuilder.standard()
-                .withCredentials(new AWSStaticCredentialsProvider(
-                        new BasicAWSCredentials(amazonAccessKeyID, amazonSecretAccessKey))
-                );
+            .withCredentials(new AWSStaticCredentialsProvider(
+                new BasicAWSCredentials(amazonAccessKeyID, amazonSecretAccessKey))
+            );
 
         if (buckets.size() > 1) {
             log.info("Using {} buckets:{}", buckets.size(), buckets);
         }
 
+
         if (StringUtils.isNullOrEmpty(amazonUrl)) {
-            amazonS3ClientBuilder = amazonS3ClientBuilder.withRegion(amazonRegion);
-            amazonMappedUrl = "s3://" + primaryBucket + "/" + bucketPath + "/";
+            amazonUrl = amazonProtocol + amazonDomain;
+        }
+        final boolean isRealAmazon = ((amazonProtocol + amazonDomain).equals(amazonUrl) || (amazonDomain).equals(amazonUrl));
```

Comment:
`isRealAmazon` -> `isAmazonS3` for clarity, as we are dealing with different S3 providers

Reply:
AmazonS3 can be Minio or Ceph too. For minio and Ceph the (old) pathStyleAccess is used. Just for the real Amazon the pathStyleAccess is not used. So this seems correct to me.